pkg/asset/installconfig/azure/client.go

```diff
@@ -274,12 +275,30 @@ func (c *Client) GetDiskEncryptionSet(ctx context.Context, subscriptionID, group
 	return &diskEncryptionSet, nil
 }
 
+// GetVMCapabilities retrieves the capabilities of an instant type in a specific region. Returns these values
+// in a map with the capability name as the key and the corresponding value.
+func (c *Client) GetVMCapabilities(ctx context.Context, instanceType, region string) (map[string]string, error) {
+	typeMeta, err := c.GetVirtualMachineSku(ctx, instanceType, region)
+	if err != nil {
+		return nil, fmt.Errorf("error connecting to Azure client: %s", err.Error())
```

Comment:
I've also seen the formatting being done as 
```suggestion
		return nil, fmt.Errorf("error connecting to Azure client: %v", err)
```
Not sure which one is more desirable